sound/soc/sof/pm.c

```diff
@@ -79,6 +81,8 @@ static int sof_resume(struct device *dev, bool runtime_resume)
 		return ret;
```

Comment:
here the state is no longer BOOT_PREPARE?

Reply:
@plbossart in both the core and pm resume callback, the appropriate state is set by the snd_sof_run_firmware() method.

Reply:
Yes, it will still be PREPARE if fw loading fails as there isnt really any particular action needed if fw loading fails.